modules/system/console/OctoberUtil.php

```diff
@@ -120,21 +125,135 @@ protected function utilSetBuild()
             return;
         }
 
-        try {
-            $build = UpdateManager::instance()->setBuildNumberManually();
-            $this->comment('*** October sets build: '.$build);
+        $this->comment('*** Detecting October CMS build...');
+
+        $build = UpdateManager::instance()->setBuildNumberManually();
+
+        if (is_null($build)) {
+            $this->error('Unable to detect your build of October CMS.');
+            exit(0);
```

Comment:
Is exit(0) recommended in CLI commands or is there a more laravel-y way of doing that? I ask because we need to avoid exit() / die() in the web code in order to support tools like swoole

Reply:
@LukeTowers i think returning should stop it no?

Reply:
I suppose that instance can be changed to a return, but the `exit(1)` calls elsewhere need to remain in order for the CLI command to actually end in an error state. I don't think Laravel has a method that kills the program like that.

Isn't Swoole a web request handler, not CLI?

Reply:
It is, was just checking if there was something similar in Laravel, like returning an error code instead of exiting. This specific use case should't actually be a problem for tools like Swoole, just like to be thinking about it in the back of our heads.

Reply:
@LukeTowers I've changed that instance to a `return`.